subsys/logging/Kconfig.backends

```diff
@@ -319,30 +326,18 @@ config LOG_BACKEND_FS
 
 if LOG_BACKEND_FS
 
-config LOG_BACKEND_FS_OUTPUT_DICTIONARY
-	bool
-	depends on LOG2
-	select LOG_DICTIONARY_SUPPORT
-	help
-	  FS backend is in dictionary-based logging output mode.
-
-choice LOG_BACKEND_FS_OUTPUT
-	prompt "FS Backend Output Mode"
-	default LOG_BACKEND_FS_OUTPUT_TEXT
+backend = FS
+backend-str = fs
+source "subsys/logging/Kconfig.template.log_format_config"
 
-config LOG_BACKEND_FS_OUTPUT_TEXT
-	bool "Text"
-	help
-	  Output in text.
+if LOG_BACKEND_FS_OUTPUT_DICTIONARY
 
 config LOG_BACKEND_FS_OUTPUT_DICTIONARY_BIN
 	bool "Dictionary (binary)"
-	depends on LOG2
-	select LOG_BACKEND_FS_OUTPUT_DICTIONARY
 	help
 	  Dictionary-based logging output in binary.
 
-endchoice
+endif #LOG_BACKEND_FS_OUTPUT_DICTIONARY
```

Comment:
There is no need for this, as this kconfig is not being used by the backend.

Reply:
@dcpleung The option "LOG_BACKEND_FS_OUTPUT_DICTIONARY_BIN"  does exist for the FS backend. Not sure what you meant here?

Reply:
This is not being used in the backend code.

Reply:
True. Removed this option from FS backend Kconfig.